src/common/io.ts

```diff
@@ -313,6 +287,40 @@ export class IOReadable implements AoIORead {
     });
   }
 
+  async getGatewayDelegates(
+    params?: { address: WalletAddress } & PaginationParams,
```

Comment:
Would be interesting if the paginationparams type were genericized to allow for specifying the allowable sortBy values for each use case.